tests/_internal/test_resolve_target.py

```diff
@@ -9,81 +9,101 @@
     NoTargetForCandidate,
     _get_all_htm_files_contents,
     resolve_target,
-    _build_target_resolver,
+    # _build_target_resolver,
```

Comment:
Should this be removed?

Reply:
https://github.com/ARMmbed/mbed-devices/pull/51/commits/baf517933386467f62605f01e19a2d2aa37d6151